core-build/gulp-core-build-sass/src/CSSModules.ts

```diff
@@ -0,0 +1,57 @@
+import * as path from 'path';
+
+import * as postcss from 'postcss';
+import * as cssModules from 'postcss-modules';
+import * as crypto from 'crypto';
+
+export interface ICSSModules {
+  /**
+   * Return a configured postcss plugin that will map class names to a
+   * consistently generated scoped name.
+   */
+  getPlugin: () => postcss.AcceptedPlugin;
+
+  /**
+   * Return the CSS class map that is stored after postcss-modules runs.
+   */
+  getCssJSON: () => Object;
+}
+
+export default class CSSModules implements ICSSModules {
+  private _classMap: Object;
+  private _rootPath: string;
+
+  constructor(rootPath?: string) {
```

Comment:
Can you leave a doc comment here mentioning what `rootPath` is expected to be?

Reply:
Done.